dev-support/docker-images/jupyter/Dockerfile

```diff
@@ -107,12 +107,16 @@ RUN cp submarine/submarine-sdk/pysubmarine/example/submarine_experiment_sdk.ipyn
     rm submarine -rf
 
 # Install latest stable qlib
-RUN pip install numpy==1.19.5 pyqlib==0.6.1
+RUN pip install numpy==1.19.5 pyqlib==0.6.1 boto3
```

Comment:
boto3 should be separated to another line with the comment "Install minio dependencies"